tests/app/authentication/test_authenticator.py

```diff
@@ -1,167 +1,153 @@
-from datetime import datetime, timedelta, timezone
+from datetime import datetime, timezone
 
+from flask import Flask
 from flask import session as cookie_session
-from mock import patch
 
 from app.authentication.authenticator import load_user, request_load_user, user_loader
-from app.data_models.session_data import SessionData
-from app.data_models.session_store import SessionStore
 from app.settings import USER_IK
-from tests.app.app_context_test_case import AppContextTestCase
-
-
-class TestAuthenticator(AppContextTestCase):  # pylint: disable=too-many-public-methods
-    def setUp(self):
-        super().setUp()
-        self.session_data = SessionData(
-            tx_id="tx_id",
-            schema_name="some_schema_name",
-            response_id="response_id",
-            period_str="period_str",
-            language_code=None,
-            launch_language_code=None,
-            survey_url=None,
-            ru_name="ru_name",
-            ru_ref="ru_ref",
-            case_id="case_id",
+
+
+def test_check_session_with_user_id_in_session(
+    app,
+    session_store,
+    session_data,
+    expires_at,
+    mock_get_session_store,
```

Comment:
Any reason not to do the patch of session store inline?. I find this confusing as we have 2 session stores here, but one is just to patch. But it does save a bit of duplication, but I think clarity is important. I think calling it `patch_get_session_store` might make it clearer on what it does.